_data/simple-icons.json

```diff
@@ -615,6 +675,11 @@
             "hex": "092E20",
             "source": "https://www.djangoproject.com/community/logos/"
         },
+        {
+            "title": "DLNA",
```

Comment:
Very low Alexa rank (~1.2mil), but seems to be popular enough as a certificate (see [their wikipedia page](https://en.wikipedia.org/wiki/Digital_Living_Network_Alliance)). I'm good with allowing this. Thoughts @ericcornelissen @davidklebanoff ?